internal/zinc-compile-core/src/main/scala/sbt/internal/inc/AnalyzingCompiler.scala

```diff
@@ -81,17 +74,22 @@ final class AnalyzingCompiler(
       log: xLogger
   ): Unit = {
     val progress = if (progressOpt.isPresent) progressOpt.get else IgnoreProgress
-    bridgeInstance(compilerBridgeClassName, log) match {
-      case (intf: CompilerInterface2, _) =>
-        intf.run(sources, changes, options, output, callback, reporter, progress, log)
-      case (bridge, bridgeClass) =>
+    val cp = classpath.map(converter.toPath)
+    val arguments = compArgs.makeArguments(Nil, cp, options).toArray
```

Comment:
How does this change relate to service-loading?

Reply:
Oh, and then I close this tab and read #871.  😆   Let's pull this change out of this PR, as it's has that `PathFactory` to consider.

Reply:
Addressed in ce80b80